8/debian-10/rootfs/opt/bitnami/scripts/libjasperreports.sh

```diff
@@ -152,10 +190,20 @@ jasperreports_configure_smtp() {
 #   None
 #########################
 jasperreports_configure_user() {
+    local db_type
+    db_type="$JASPERREPORTS_DATABASE_TYPE"
+
     info "Configuring users"
+
     # Change the default user username and mail using the database
-    mysql_remote_execute "$JASPERREPORTS_DATABASE_HOST" "$JASPERREPORTS_DATABASE_PORT_NUMBER" "$JASPERREPORTS_DATABASE_NAME" "$JASPERREPORTS_DATABASE_USER" "$JASPERREPORTS_DATABASE_PASSWORD" <<<"UPDATE JIUser SET username='${JASPERREPORTS_USERNAME}' WHERE id=1"
-    mysql_remote_execute "$JASPERREPORTS_DATABASE_HOST" "$JASPERREPORTS_DATABASE_PORT_NUMBER" "$JASPERREPORTS_DATABASE_NAME" "$JASPERREPORTS_DATABASE_USER" "$JASPERREPORTS_DATABASE_PASSWORD" <<<"UPDATE JIUser SET emailAddress='${JASPERREPORTS_EMAIL}' WHERE id=1"
+    if [[ "$db_type" = "postgresql" ]]; then
+        postgresql_remote_execute "$JASPERREPORTS_DATABASE_HOST" "$JASPERREPORTS_DATABASE_PORT_NUMBER" "$JASPERREPORTS_DATABASE_NAME" "$JASPERREPORTS_DATABASE_USER" "$JASPERREPORTS_DATABASE_PASSWORD" <<<"UPDATE JIUser SET username='${JASPERREPORTS_USERNAME}' WHERE id=1"
+        postgresql_remote_execute "$JASPERREPORTS_DATABASE_HOST" "$JASPERREPORTS_DATABASE_PORT_NUMBER" "$JASPERREPORTS_DATABASE_NAME" "$JASPERREPORTS_DATABASE_USER" "$JASPERREPORTS_DATABASE_PASSWORD" <<<"UPDATE JIUser SET emailAddress='${JASPERREPORTS_EMAIL}' WHERE id=1"
```

Comment:
In order to simplify the logic a bit, we could merge the two SQL queries into one command, e.g.:
```suggestion
        postgresql_remote_execute "$JASPERREPORTS_DATABASE_HOST" "$JASPERREPORTS_DATABASE_PORT_NUMBER" "$JASPERREPORTS_DATABASE_NAME" "$JASPERREPORTS_DATABASE_USER" "$JASPERREPORTS_DATABASE_PASSWORD" <<EOF
UPDATE JIUser SET emailAddress='${JASPERREPORTS_EMAIL}' WHERE id=1;
UPDATE JIUser SET username='${JASPERREPORTS_USERNAME}' WHERE id=1;
EOF
```

Otherwise it would be a bit difficult to read. Could you also update the logic for the MySQL case as well?